inst/NORIC_local_monthly.Rmd

```diff
@@ -1,19 +1,20 @@
 ---
-date: '`r format(Sys.time(), "%d %B, %Y")`'
+title: "`r paste('Norsk register for invasiv kardiologi (NORIC)', 'MÃ¥nedsrapport Angio/PCI for ', params$hospitalName ) `"
```

Comment:
Merkelig tegn for "å". Skyldes sikkert tegnsettet som benyttes på din datamaskin...

Reply:
> > > Se kommentarer i koden. Ellers veldig bra ryddejobb!
> > 
> > 
> > Hvordan ser jeg kommentarene i koden ?
> 
> Rett over, i denne tråden...

Ang dato og tittel ? Det har eg sett !

Reply:
Jeg har satt inn en forståelig "å" :-)